net/samba4/Portfile

```diff
@@ -75,7 +76,9 @@ configure.perl      ${perl5.bin}
 configure.python    ${prefix}/bin/python3.10
 configure.env-append \
                     YAPP=${prefix}/bin/yapp-${perl5.major}
-configure.cmd       ${configure.python} ./buildtools/bin/waf configure
+# provided by samba configure script actually does this, but setting environment correctly.
+# bypassing it breaks build on ppc: https://trac.macports.org/ticket/65622
```

Comment:
Your grammar is a bit less-than-comprehensible here. Indeed, I'm not sure what you're saying...

Reply:
Sorry, not a native language for me.

samba4 has a configure script, which redirects to `./buildtools/bin/waf`, but it also sets environment. Portfile for an unclear reason bypasses native configure, and that breaks ppc (it freezes on configure, which I reported on Trac).
I will rephrase now.

Or should I just drop the comment from there?

Reply:
Amended the comment, hopefully it is acceptable now :) Rebased to master.